### ISSUE TITLE
Tooltip: Add Maximum Length

### ISSUE THREAD

**Issue by JeremyVyska** [MicrosoftDocs/dynamics365smb-devitpro-pb#2267]
[Enter feedback here]

One thing this page could use:  What is the maximum length a ToolTip may have?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ffd74fd6-548d-095d-34c6-0c3b7b00d22e
* Version Independent ID: 298b8ca2-450f-0015-5fdb-001cae86124b
* Content: [ToolTip Property - Business Central](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/properties/devenv-tooltip-property)
* Content Source: [dev-itpro/developer/properties/devenv-tooltip-property.md](https://github.com/MicrosoftDocs/dynamics365smb-devitpro-pb/blob/main/dev-itpro/developer/properties/devenv-tooltip-property.md)
* Service: **dynamics365-business-central**
* GitHub Login: @SusanneWindfeldPedersen
* Microsoft Alias: **solsen**

**Comment by edupont04** [comment]
Hi Jeremy,
We have a best practice for tooltips to not exceed 200 characters: https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/user-assistance#guidelines-for-tooltip-text
However, as far as I know, tooltips can technically be far longer. How long do you want it to be?

**Comment by JeremyVyska** [comment]
I needed to know the specific limit as it is, as one customer was looking to add some long details to their PTE pages.

Turns out, my partner discovered this:
-	The limit is set to 1024 characters
-	If you exceed this limit, it will only show 1023 characters, with a (…) at the end to indicate there is more written, but this won't be able to be shown.

There shouldn't be any need to change this, but it'd be good to mention "We recommend you use 200 or less, but this can support up to 1024.  Tooltips over 1024 will not cause an error, but will not display.".
